### PR TITLE
AppLocker PowerShell reference update

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
[1871144](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1871144)

I updated an example to use the new param. I didn't follow guidance on omitting the prompt (or make any other changes to the example) for consistency with the other two examples.

### CODE REVIEW

**Comment by TimShererWithAquent** [comment]
Getting feedback on new text now.

**Comment by TimShererWithAquent** [comment]
Text approved by product team in email. Ready for review.

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review.

**Comment by TimShererWithAquent** [comment]
@tiburd - This is ready for review.

**Review by ghost — APPROVED** [review]

File: docset/winserver2022-ps/applocker/New-AppLockerPolicy.md [hunk 1]
```diff
@@ -74,14 +87,41 @@ The existing AppLocker policy in the specified GPO will be overwritten.
 
 ## PARAMETERS
 
+### -AllowWindows
+Indicates that the AppLocker policy allows all local Windows components.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: FileInformation
```
Comment (get-itips) [inline docset/winserver2022-ps/applocker/New-AppLockerPolicy.md:95]: I am not sure if adding two yaml blocks under the parameter is the correct way or specifying

`Parameter Sets: (All)` 

like other cmdlet articles do, I do now that one specifies required true and the other false... @dariomws @tseward ?

**Review by get-itips — CHANGES_REQUESTED** [review]
as my comment

File: docset/winserver2022-ps/applocker/New-AppLockerPolicy.md (hunk 1 above)
Comment (dariomws) [reply docset/winserver2022-ps/applocker/New-AppLockerPolicy.md:95]: I agree. Maybe the best way is using `Parameter Sets: (All)`. And looking over the other example it seems that this parameter is not required.

**Review by dariomws — COMMENTED** [review]
Agree with @get-itips.

**Review by dariomws — APPROVED** [review]

**Pull request merged**